### PR TITLE
projectlibre: init at 1.6.2

### PR DESCRIPTION
**Pull request by mogria**
###### Motivation for this change

Alternative to ganttproject with more features. More similar to MS-Project.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/projectlibre/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    cp -R openproj_build/dist/* $out/share/projectlibre
+    cp $desktopItem $out/share/applications
+    cp $resourcesPath/projectlibre.png $out/share/pixmaps
+    cp -R $resourcesPath/samples/* $out/share/projectlibre/samples
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "http://www.projectlibre.com/";
+    descripton = "Project-Management Software similar to MS-Project";
+    maintainer = with maintainers; [ mogria ];
```
Comment (ndowens) [inline pkgs/applications/misc/projectlibre/default.nix]: Use maintainer.mogria instead as with maintainers is used for more than one maintainers

**Comment by joachifm** [comment]
Travis fails with
> building path(s) ‘/nix/store/8rm3nlh7zniz8vb5p6ajqhgwsg81lzzk-code’
exporting https://git.code.sf.net/p/projectlibre/code (rev v1.6.2) into /nix/store/8rm3nlh7zniz8vb5p6ajqhgwsg81lzzk-code
Bad commit hash or bad reference.

I believe it's because `v` is prepended to the revision passed to `fetchgit`.

File: pkgs/applications/misc/projectlibre/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchgit, ant, jdk, jre }:
+
+stdenv.mkDerivation rec {
+  name = "projectlibre-${version}";
+  version = "1.6.2";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/projectlibre/code";
+    rev = "v${version}";
```
Comment (joachifm) [inline pkgs/applications/misc/projectlibre/default.nix]: Maybe try something like `refs/tags/v${version}` instead

File: pkgs/applications/misc/projectlibre/default.nix (hunk 1 above)
Comment (mogria) [inline pkgs/applications/misc/projectlibre/default.nix]: should I push another commit or force push to my branch?

**Comment by mogria** [comment]
Is there still any issue with this?

**Comment by vcunat** [comment]
There are minor problems in `meta` (descript**i**on, `maintainers = [ maintainers.mogria ]`),

but the main thing is that it won't start unless you have some java binaries on $PATH:
```
$ ./result/bin/projectlibre 
Java auto-detection...
Checking java
    Error
Java not found or incorrect version.
Please install OpenJDK, Oracle JRE 1.6+ or set JAVA_HOME environment variable if it's already installed.
```

**Comment by vcunat** [comment]
Well, even if I do that, the displayed window is strangely empty &ndash; I created a new project and tried adding a few items, but all I'm getting is an all-gray window without anything really displayed.

**Comment by joachifm** [comment]
So, does this work as packaged or not?

**Comment by bjornfor** [comment]
Running with empty PATH reveals some missing dependencies:

```
$ PATH= ./result/bin/projectlibre 
./result/bin/projectlibre: line 85: uname: No such file or directory
./result/bin/projectlibre: line 171: java: No such file or directory
Java auto-detection...
Checking java
./result/bin/projectlibre: line 124: rm: No such file or directory
./result/bin/projectlibre: line 127: cat: No such file or directory
./result/bin/projectlibre: line 127: awk: No such file or directory
    Java version:  ./result/bin/projectlibre: line 136: sed: No such file or directory
./result/bin/projectlibre: line 137: sed: No such file or directory
./result/bin/projectlibre: line 137: awk: No such file or directory
NOK
Java not found or incorrect version.
Please install OpenJDK, Oracle JRE 1.6+ or set JAVA_HOME environment variable if it's already installed.
```

Also, the path to java seems to be stored in ~/.projectlibre/run.conf. I'm not sure if that path gets updated on every run, but just in case, I'd pass only program names to projectlibre, instead of full nix store paths. (Or else things may break when updating / garbage collecting.) 

Injecting ```export "PATH=${jre}/bin:${coreutils}/bin:${which}/bin:\$PATH"``` into the startup script should suffice. (Adding "impure" $PATH to the end because projectlibre may want to start your $BROWSER.)

Lastly, @mogria, would you mind interactively rebasing the PR so that the commit sequence is:
1. add license
2. add maintainer
3. add package

Thanks!

**Comment by bjornfor** [comment]
And please update to version 1.7.0.

**Comment by bjornfor** [comment]
@mogria: Ping?

**Pull request closed without merging**